### ISSUE TITLE
small chicken issue

### ISSUE THREAD

**Issue by fabznn** [blizzhackers/kolbot#326]
there is one small issue with chicken:

when you are using a druid with Oak Sage, sometimes the game bugs and when a char gets the life bonus, it gets chickened out of the game. i was using chicken = 50.
i will try it now with standard chicken = 30.

maybe there is a way to solve this issue anyways.

**Comment by aim2kill** [comment]
Yes, If he precast the oaksage when he at 500/1000 hp(under 50%) he hp will end up under the threshold and quick

**Comment by theBGuy** [comment]
try using the [chicken-patch](https://github.com/blizzhackers/kolbot/tree/chicken-patch) branch and let me know if this goes away
